clouddriver-cf/src/main/groovy/com/netflix/spinnaker/clouddriver/cf/provider/view/CatsApplicationProvider.groovy

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2016 Pivotal, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.cf.provider.view
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.netflix.spinnaker.cats.cache.Cache
+import com.netflix.spinnaker.cats.cache.CacheData
+import com.netflix.spinnaker.cats.cache.RelationshipCacheFilter
+import com.netflix.spinnaker.clouddriver.cf.CloudFoundryCloudProvider
+import com.netflix.spinnaker.clouddriver.cf.cache.Keys
+import com.netflix.spinnaker.clouddriver.cf.model.CloudFoundryApplication
+import com.netflix.spinnaker.clouddriver.model.Application
+import com.netflix.spinnaker.clouddriver.model.ApplicationProvider
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+
+import static com.netflix.spinnaker.clouddriver.cf.cache.Keys.Namespace.APPLICATIONS
+import static com.netflix.spinnaker.clouddriver.cf.cache.Keys.Namespace.CLUSTERS
+/**
+ * @author Greg Turnquist
+ */
+@Component
+class CatsApplicationProvider implements ApplicationProvider {
```

Comment:
I'd be ok with just calling this CloudFoundryApplicationProvider (and I would rename CatsApplicationProvider in the aws module at some point)